wandb/board/ui/src/util/apollo.js

```diff
@@ -99,7 +99,13 @@ const userTimingMiddleware = new ApolloLink((operation, forward) => {
   });
 });
 
+const graphqlStatusMiddleware = new ApolloLink((operation, forward) => {
+  dispatch(updateGraphqlStatus(true));
```

Comment:
Ok, I kinda understand what you're trying to do here, but don't think this is the place to do it.  If graphqlStatus actually means network status, it's not true that we are connected to the network at this point.  It will result in graphqlStatus being true momentarily before the error link sets it to false.  The only place you check graphql status is in the history writer, why is that?  Do we really need this global state?

Reply:
Idea was to have graphql check, and not to be the state of network.